content/en/news/releases/1.18.x/announcing-1.18.3/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Announcing Istio 1.18.3
+linktitle: 1.18.3
+subtitle: Patch Release
+description: Istio 1.18.3 patch release.
+publishdate: 2023-09-12
+release: 1.18.3
+---
+
+This release contains bug fixes to improve robustness.
+
+This release note describes what’s different between Istio 1.18.2 and 1.18.3.
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** ability to install gateway helm chart with a dual-stack service definition.
+
+- **Fixed** app probe: http `request.host` is not well propagated.
+  ([Issue #46087](https://github.com/istio/istio/issues/46087))
+
+- **Fixed** `health_checkers` extensions is not compiled in.
+  ([Issue #46277](https://github.com/istio/istio/issues/46277))
+
+- **Fixed** an issue that Istio should using `IMDSv2` as possible on AWS.
```

Comment:
```suggestion
- **Fixed** an issue that Istio should use `IMDSv2` if possible on AWS.
```

Reply:
Need to verify that I corrected that. Is this a new default, or is this just a new possible value?

Reply:
I see a comment in that PR (not changed):
```
**Fixed** an issue that istio running on AWS should prefer IMDSv2 over IMDSv1
```